proto/swagger/serve.swagger.json

```diff
@@ -799,12 +855,16 @@
     "protobufAny": {
       "type": "object",
       "properties": {
-        "@type": {
+        "typeUrl": {
```

Comment:
This is breaking api change, is it necessary?

Reply:
This is autogenerated, it is version 1.19 vs 1.17